biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -486,7 +489,12 @@
 	/*
 	 * Module Section
 	 */
+	String								MODULE										= "-module";
+	String								EXPORT_TO_MODULES_DIRECTIVE					= "-export-to-modules:";
```

Comment:
As mentioned above, can we not define a jpms annotation to do this instead of a directive?